Lib/datetime.py

```diff
@@ -925,7 +925,33 @@ def ctime(self):
             self._day, self._year)
 
     def strftime(self, fmt):
-        "Format using strftime()."
+        """strftime(fmt) -> date_string
+
+        Return a string from the date parsed according to format.
+        Format codes referring to hours, minutes or seconds will see 0 values.
+
+        Args:
+            format: representation of date_string using format codes.
```

Comment:
1. The argument is named "fmt".
2. The "Args:" style is different from that currently used in these classes, and adds little in this case with just one argument.
3. I'm not sure "representation of date_string using format codes" would be clear to someone who hadn't seen this before. It seems to me that an example would be very helpful here.